mandatory/3-playing-computer.js

```diff
@@ -40,3 +40,18 @@ for (let i = 0; i < 5; ++i) {
     console.log(e);
   }
 }
+// 1. This program throws an error. Why? (If you can't find it, try executing it).
+// x is not defined
+// 2. Remove the line that throws the error.
+// line 18
+// 3. What is printed to the console?
+//
+// 4. How many times is "f1" called?
+// 2
+// 5. How many times is "f2" called? 
+// 2
```

Comment:
This isn't quite right, have another look at the loop and double check how many times it loops, remembering that it starts at 0